### PR TITLE
Some various small fix

### PR DESCRIPTION
**Pull request by samuelgarcia**
Some various small fix that prement me to make a clear demo on spikeinterface.

 * improve blackrock speed (depend on https://github.com/NeuralEnsemble/python-neo/pull/1231) by skiping the nev
 * auto select the blacrock stream id (nx2 o, nx6, nx2) using the file suffix
 * use neo channel_names optionaly when possible
 * improve plot_probe_map when sevral probes

### CODE REVIEW

**Comment by alejoe91** [comment]
We should merge this first: #1410

**Comment by samuelgarcia** [comment]
The neo side have been merged.

File: spikeinterface/extractors/neoextractors/neobaseextractor.py [hunk 1]
```diff
@@ -78,8 +84,14 @@ def __init__(self, stream_id=None, stream_name=None,
         mask = signal_channels['stream_id'] == stream_id
         signal_channels = signal_channels[mask]
 
-        # check channel groups
-        chan_ids = signal_channels['id']
+        if use_names_as_ids :
```
Comment (alejoe91) [inline spikeinterface/extractors/neoextractors/neobaseextractor.py:87]:
```suggestion
        if use_names_as_ids:
```

File: spikeinterface/extractors/neoextractors/neobaseextractor.py [hunk 2]
```diff
@@ -109,7 +121,8 @@ def __init__(self, stream_id=None, stream_name=None,
 
         self.set_property('gain_to_uV', final_gains)
         self.set_property('offset_to_uV', final_offsets)
-        self.set_property('channel_name', signal_channels["name"])
+        if not use_names_as_ids :
```
Comment (alejoe91) [inline spikeinterface/extractors/neoextractors/neobaseextractor.py:124]:
```suggestion
        if not use_names_as_ids:
```

**Comment by samuelgarcia** [comment]
If pass, ready to reveiw on my side.

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**